client/network/src/config.rs

```diff
@@ -445,6 +461,67 @@ where
 	fs::OpenOptions::new().write(true).create_new(true).open(path)
 }
 
+/// The configuration of a node's WebRTC certificate, describing how it is obtained.
+#[derive(Clone, Debug)]
+pub enum WebRTCConfig {
+	/// A WebRTC certificate.
+	Certificate(Option<PathBuf>),
```

Comment:
```suggestion
	FromDisk(Option<PathBuf>),
```

?

If you name this `Certificate`, I don't really see what the other variants would be